### PR TITLE
Revamp Spotlight state landing page

### LINKED ISSUE #476: Revamp Spotlight state landing page
**Is your feature request related to a problem? Please describe.**
The current Spotlight homepage - for both North Dakota and Pennsylvania - is very boring and leads to unacceptably high bounce rates. We want to reduce those bounce rates to ensure the app achieves its mission of increased transparency and information sharing.

**Describe the solution you'd like**
Update the Spotlight landing page ([current US_ND](https://dashboard.docr.nd.gov/us-nd), [current US_PA](https://dashboard.cor.pa.gov/us-pa)) to match what is described in the "Spotlight — Home Page Update" 1-pager and associated mocks. This includes:
- Updated top text styling and copy
- Display of charts directly on the landing page (these are all charts that are already displayed elsewhere in the app and simply need to be composed here, too)
- Testing across browsers and testing in mobile

**Describe alternatives you've considered**
N/A

**Additional context**
See the 1-pager and mocks for more details.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Revamp Spotlight state landing page

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #476 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by jessex — COMMENTED** [review]
Holding off on a more thorough review for now, since there's not a whole lot to comment on just yet!

**Comment by nasaownsky** [comment]
@jessex Hey, Josh! So I have something like this for now: 

screencapture-localhost-3000-us-nd-2021-08-24-20_02_16

@cawarren suggested using other charts to preview sentencing and parole data and they are much easier to manage.

The solution for the preview is using a boolean flag, depending on which we render different charts. The only thing I need to do is to get rid of prison population minimap. Maybe @macfarlandian know simple and effective solution for that?

**Comment by macfarlandian** [comment]
@nasaownsky hmm tricky .... I don't think that can be disabled, probably you will have to sub out the `MinimapXYFrame` component for a plain `XYFrame` in preview mode. It should take all of the same props with the exception of `minimap` so hopefully that won't be too bad

**Comment by nasaownsky** [comment]
@jessex Hey! In case you have missed this - take a look, it should be ready to go!

**Review by jessex — APPROVED** [review]
This is looking good to me! Thanks so much for this, @nasaownsky. Functionality looks solid in local testing, UI looks like the mocks, and I have no more code comments.

@macfarlandian, would you mind taking a pass through this this week? Ilya has shifted back to some final pre-launch updates for the Cost Calculator for the rest of this week so you have some time. But since this has some non-trivial modeling updates, I want to make sure you're on board before we merge.

File: spotlight-client/src/contentModels/types.ts
Comment (macfarlandian) [inline spotlight-client/src/contentModels/types.ts:93]: per comments about the links above, I don't think we are going to want to make this change in the end.

File: spotlight-client/src/charts/BubbleChart.tsx
Comment (macfarlandian) [inline spotlight-client/src/charts/BubbleChart.tsx:75]: This is a design rather than an implementation issue, but I don't actually think we should be hiding the legend on this preview. The chart is totally unreadable without it. We include the legend for other preview charts where needed, so there is precedent for this. I will appeal to @hobuobi for confirmation on this, though.

Failing that, could we call this prop "showLegend" or something instead? "preview" is not really meaningful in this context, the charts in this directory are meant to be as generic as possible.

File: spotlight-client/src/charts/WindowedTimeSeries.tsx
Comment (macfarlandian) [inline spotlight-client/src/charts/WindowedTimeSeries.tsx:231]: I think by doing it this way we are creating a bunch of expensive React and Semiotic stuff that we don't intend to render, as well as defining most of the props redundantly? 

Rather than duplicating all these props, we should just collect them in a base object that we extend as needed with the additional props needed for the minimap. And instead of instantiating both Frame components we should just conditionally pick one ... could either store that as a variable or conditionally render `{preview ? <XYFrame {...chartProps} /> : <MinimapXYFrame {...chartProps} />}` instead of what's currently there.

File: spotlight-client/src/charts/WindowedTimeSeries.tsx
Comment (macfarlandian) [inline spotlight-client/src/charts/WindowedTimeSeries.tsx:107]: as I have commented elsewhere, I'm not really a fan of "preview" in this interface, which is meant to be more generic. I think if we instead had props for like, `showMinimap` and `showLegend`, then the parent could pass those as needed to create the desired "preview" effect.

**Review by macfarlandian — CHANGES_REQUESTED** [review]

File: spotlight-client/src/charts/BubbleChart.tsx
Comment (hobuobi) [reply spotlight-client/src/charts/BubbleChart.tsx:75]: I tend to agree! No need to get rid of the legend.

**Comment by macfarlandian** [comment]
realized I forgot to post one other thing ... I noticed that the tooltips can be cut off in the "preview" charts if they are close to the right edge. example screenshots: 

Screen Shot 2021-08-31 at 5 05 47 PM

Screen Shot 2021-08-31 at 5 05 40 PM

**Comment by nasaownsky** [comment]
Hey, @macfarlandian! So I've made all the changes you have requested. Check that out!

File: spotlight-client/src/SystemNarrativePage/constants.ts
Comment (macfarlandian) [inline spotlight-client/src/SystemNarrativePage/constants.ts:18]: can delete this eslint override now that there is more than one export

**Review by macfarlandian — COMMENTED** [review]
getting very close, @nasaownsky ! other than a couple of nits I commented on here, the main remaining issue is with the navigation links at the bottom of the narrative pages. 

If you were to look at, e.g., [this page](https://dashboard.docr.nd.gov/us-nd/narratives/prison/8) on the live site, it still looks a little different than it does with these changes: 
Screen Shot 2021-09-08 at 9 59 42 AM

The layout is a bit tighter (can fit four to a row) and the labels don't say "Data" like they do on the new homepage design. We still want the links on those pages to remain unchanged from what they were before, so I think there needs to be even more separation between the components than you have here. And just to give you a heads-up, there is a further change coming (still being spec'ed out, will be a separate ticket) that will include the Racial Disparities link on the homepage but also reshuffle the order, so the functionality of the homepage versus the other pages will diverge even more. I don't think there's going to be much benefit in trying to adapt one component to render both versions, so if breaking the homepage links out into a whole new component makes this easier then you should feel free to make that change now

**Comment by nasaownsky** [comment]
@macfarlandian ok, I've added brand new component `OtherNarrativeLinksPreview` and since the Racial Disparities not included on the homepage, there is no need to modify types and extract `NarrativeLink` from both `OtherNarrativeLinksPreview` and `OtherNarrativeLinks` as its rely on different types and there is no specification for Racial Disparities for now.

File: spotlight-client/src/contentApi/sources/us_nd.ts
Comment (macfarlandian) [inline spotlight-client/src/contentApi/sources/us_nd.ts:425]: looks like a typo here, we want users to see quotation marks around "Free Through Recovery", i.e. `"Free Through Recovery" Program, Participants Statewide`

File: spotlight-client/src/contentApi/sources/us_nd.ts
Comment (macfarlandian) [inline spotlight-client/src/contentApi/sources/us_nd.ts:314]: in ND this data actually includes probation also, so this should say "Prison or Probation"

File: spotlight-client/src/contentApi/sources/us_pa.ts
Comment (macfarlandian) [inline spotlight-client/src/contentApi/sources/us_pa.ts:119]: similarly, we are not showing the age breakdowns in this preview, so it should say "by Reason" instead

File: spotlight-client/src/contentApi/sources/us_pa.ts
Comment (macfarlandian) [inline spotlight-client/src/contentApi/sources/us_pa.ts:46]: we are not actually showing the race/ethnicity breakdown in the preview so this should also just say "Prison Population over Time" like it does in ND

**Review by macfarlandian — COMMENTED** [review]
did one more QA pass and noticed that some of the preview labels didn't match the data we were showing ... noted them in the comments here. I think that should be the last thing on this PR

**Comment by nasaownsky** [comment]
Done!

**Review by macfarlandian — APPROVED** [review]
🚀

**Pull request merged**
